### PR TITLE
Clawdit

### PR DESCRIPTION
**Pull request by rangel35**
**GitHub Issue**: (Islandora-CLAW/CLAW#1267)

# What does this Pull Request do?

Update to Readme.md to follow Islandora 8 template and update maintainers

# Interested parties
Tag (@Islandora-CLAW/committers)

### CODE REVIEW

File: README.md
Comment (manez) [inline README.md]: Should be bullets, not a single line

File: README.md
Comment (manez) [inline README.md]: The logo seems a big big. maybe sub is a smaller version? 
image

**Review by manez — CHANGES_REQUESTED** [review]

File: README.md
Comment (rangel35) [reply README.md]: this is the logo on the repo...I can try to shrink it and drop in a smaller version...I have Photoshop :)

File: README.md
Comment (rangel35) [reply README.md]: done...

File: README.md
Comment (rangel35) [reply README.md]: thanks for the link...I didn't realize that was the link until I actually thought about it so I used yours

File: README.md
Comment (manez) [inline README.md]: Extra C in LICENSE is breaking the link

/me takes three tries to spell license properly when pointing this out...

**Review by manez — CHANGES_REQUESTED** [review]

File: README.md
Comment (rangel35) [reply README.md]: someday my english will catchup to my gibberish and I will be fully fluent in both

**Review by manez — DISMISSED** [review]

**Comment by rosiel** [comment]
I came across another module that had been failing Travis, and @whikloj figured out why... 
I think that because of some changes in Travis since this module was last updated, you need to add 

```
services:
  - mysql
```
to the .travis.yml file.

See for example  https://github.com/Islandora-CLAW/islandora/compare/8.x-1.x...whikloj:travis-fix#diff-354f30a63fb0907d4ad57269548329e3R6-R7

**Comment by rangel35** [comment]
added to travis.yml
services:
  - mysql

**Comment by dannylamb** [comment]
This build is failing for the same reasons https://github.com/Islandora-CLAW/carapace/pull/11 was.  I've fixed it in that PR, so we can merge this once it lands.

File: README.md
Comment (whikloj) [inline README.md]: As this PR got hung up during the paged content merge, now it requires some more changes. But I want @dannylamb to just confirm my statements.

Statement 1: We no longer require `at_tools`

Comment (whikloj) [reply README.md]: Statement 1.1: The below lines 23-26 can be removed as they refer to `at_tools`

File: README.md
Comment (whikloj) [inline README.md]: Remove `and enabled the associated module` as this is a reference to `at_tools`

**Review by whikloj — CHANGES_REQUESTED** [review]
@rangel35 the ground shifted under your PR, but I think these changes would get us to correct if I am correct that at_tools is no longer a requirement. @dannylamb?

**Comment by rangel35** [comment]
> @rangel35 the ground shifted under your PR, but I think these changes would get us to correct if I am correct that at_tools is no longer a requirement. @dannylamb?

@dannylamb verify please and I will make the requested removals from the readme

File: README.md
Comment (rangel35) [reply README.md]: removed at_tools references

**Review by whikloj — APPROVED** [review]
👍 Looks good to me.

**Review by manez — APPROVED** [review]

**Comment by dannylamb** [comment]
Sorry this one slipped by me.  @rangel35 was correct in removing the blip about dependencies.  If you install with composer there's nothing else to do.

**Pull request merged**
